src/Facility.Definition/Fsd/FsdParsers.cs

```diff
@@ -182,7 +182,9 @@ internal static class FsdParsers
 		private static string TryParseAttributeParameterValue(Match match) =>
 			match.Groups[1].Success
 				? string.Concat(match.Groups[2].Captures.OfType<Capture>().Select(x => x.ToString()).Select(x => x[0] == '\\' ? DecodeBackslash(x) : x))
-				: match.Groups[3].ToString();
+				: match.Groups[3].Success
+					? match.Groups[3].Value
```

Comment:
```suggestion
					? string.Concat(match.Groups[4].Captures.OfType<Capture>().Select(x => x.ToString()).Select(x => x[0] == '\\' ? x.Substring(1) : x))
```

This would treat regex syntax as merely an alternative way to encode a string, which seems fine with me.

Reply:
Seems like the slashes require escaping again with this change. The value of the slash regex syntax was originally that they wouldn't require that.

Reply:
Oh, right! I spaced on that. `match.Groups[3].Value` is correct.